Chains/multishine.py

```diff
@@ -39,6 +39,7 @@ def step(self, gamestate, smashbot_state, opponent_state):
             self.interruptible = False
             return
 
+        #jcstates = [Action.DOWN_B_GROUND, Action.DASHING, Action.RUNNING]
```

Comment:
Looks like this line is commented out. Should it be? Just delete it if it's not needed.

Reply:
I actually intended to insert jcstates into the next line, i.e.

if smashbot_state.action in jcstates: (changed from == Action.DOWN_B_GROUND)

but I forgot to to do so.

I believe I did this because I ran into an instance where Chains.Multishine got called while Fox was still in his dash animation, causing him to get confused and do nothing. However, upon further consideration, he should be able to hit downB while running to shine anyway.

Also, upon further consideration, it will probably be a good idea to add a few more shineable states at the beginning of the file as well, since it only lists STANDING and TURNING atm, but I think it would make sense to add RUNNING, RUN_BRAKE, CROUCH_START, CROUCHING, CROUCH_END etc.